services/commands_automator_service.py

```
@@ -1,159 +1,178 @@
+
+
+class CommandsAutomatorService:
+    def __init__(self):
+        self.names_to_scripts = {}
+        self.scripts_attributes = {}
+        self.load_scripts_config()
+
+
+    def load_scripts_config(self):
+        try:
+            f = open('config\\scripts_config.json')
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)